package.json

```diff
@@ -16,7 +16,8 @@
     "build": "node webpack/build.js",
     "test": "node --max-old-space-size=4096 node_modules/.bin/karma start",
     "test:watch": "node --max-old-space-size=4096 node_modules/.bin/karma start --no-single-run --auto-watch",
-    "lint": "eslint '{src,config}/**/*.{js,vue}' --cache --fix --max-warnings=0"
+    "lint": "eslint '{src,config}/**/*.{js,vue}' --cache --fix --max-warnings=0",
+    "lint:css": "stylelint 'src/**/*.{vue,css,scss}' --syntax scss"
```

Comment:
We can not use the `--fix` flag because of [this](https://github.com/tokend/web-client/pull/51/files#diff-2f92742cff840d84253c69eb0ae8f097R6) – `processors` is blocking the possibility to use the auto fix. But we can't refuse it because of `processors` enable Stylelint to extract styles from within non-stylesheet files (in our case this is a `.vue` files).

Reply:
It still shows wrong line number with error, so I chose way with manual errors fixing but with right displaying of the line with an error.

I think we can use double or dynamic configs for it: use processors in dev-mode to show correct line with an error and clean up processors with adding `fix: true` rule in pre-commit/push/deploy hooks. What do you think about it? I need to research how to implement dynamic config, but I think it's possible